packages/ui/src/components/skeletons/skeleton-typography.tsx

```diff
@@ -0,0 +1,45 @@
+import { forwardRef } from 'react'
+
+import { cn } from '@utils/cn'
+import { cva, VariantProps } from 'class-variance-authority'
+
+import { SkeletonBase } from './components/skeleton'
+
+const skeletonTypographyVariants = cva('cn-skeleton-typography-wrapper', {
+  variants: {
+    variant: {
```

Comment:
What about importing textVariants from Text component file here? If a new variants added, they will be extended automatically here